api/capi/src/nntrainer.cpp

```diff
@@ -739,6 +739,13 @@ int ml_train_layer_set_property(ml_train_layer_h layer, ...) {
   return status;
 }
 
+int ml_train_layer_set_property_with_single_param(ml_train_layer_h layer,
```

Comment:
Do we need to make another function? It seems like it can be possible to directly use ml_train_layer_set_property I guess.

Reply:
Discussed offline.